File: SmartCom/SmartCom/app/model/Item.js
//модель товара
Ext.define('ExtMVC.model.Item', {
    extend: 'Ext.data.Model',
    idProperty: 'ItemID',

    fields: [
        { name: 'ItemID', type: 'int' },
        { name: 'Title', type: 'string' },
        { name: 'Level', type: 'int' },
        { name: 'CategoryID', type: 'int' },
        { name: 'BrandID', type: 'int' },
        ]    
});

File: SmartCom/SmartCom/Models/Item.cs
using SmartCom.Areas.Brand.Models;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

//класс товара
namespace SmartCom.Models
{
    public class Item
    {
        [Key]
        public int ItemID { get; set; } 
        public int Level { get; set; }
        public string Title { get; set; }
        public int CategoryID { get; set; }
        public int? BrandID { get; set; }
        
        //public virtual Brand Brand { get; set; }
        public virtual Category Category { get; set; }

    }
}